receiver/telemetrygeneratorreceiver/config.go

```diff
@@ -0,0 +1,117 @@
+// Copyright observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package telemetrygeneratorreceiver generates telemetry for testing purposes
+package telemetrygeneratorreceiver //import "github.com/observiq/bindplane-agent/receiver/telemetrygeneratorreceiver"
+
+import (
+	"errors"
+	"fmt"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+)
+
+// Config is the configuration for the telemetry generator receiver
+type Config struct {
+	PayloadsPerSecond int               `mapstructure:"payloads_per_second"`
+	Generators        []GeneratorConfig `mapstructure:"generators"`
+}
+
+// GeneratorConfig is the configuration for a single generator
+type GeneratorConfig struct {
+	// Type of generator to use, either "logs", "host_metrics", or "windows_events"
+	Type generatorType `mapstructure:"type"`
+
+	// ResourceAttributes are additional key-value pairs to add to the resource attributes of telemetry.
+	ResourceAttributes map[string]string `mapstructure:"resource_attributes"`
```

Comment:
Resource attributes should also be a `map[string]any` they are the same type I believe in pdata.